x/tokenfactory/genesis.go

```diff
@@ -0,0 +1,53 @@
+package tokenfactory
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v7/x/tokenfactory/keeper"
+	"github.com/osmosis-labs/osmosis/v7/x/tokenfactory/types"
+)
+
+// InitGenesis initializes the capability module's state from a provided genesis
+// state.
```

Comment:
```suggestion
// InitGenesis initializes the token factory module's state from a provided genesis
// state.
```

Reply:
Also I think we should move this all to the keeper in a second PR (and do the same for all of our modules)

This has caused several functions to be publicly exported that otherwise ought to be kept private

Reply:
New issue for that made #1402